examples/Loading and preprocessing DSGE data.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>There's a helper function in <code>gEconpy.plotting</code> called <code>prepare_gridspec_figure</code>,it sets up arrays of subplots for non-square numbers of plots, and will prevent you from having these two empty ones at the end. You can see how it is used here: https://github.com/jessegrabowski/gEconpy/blob/cd2ae75747b0aa8bc23a6b35033f3a3e4ca7bc79/gEconpy/plotting/plotting.py#L237   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>